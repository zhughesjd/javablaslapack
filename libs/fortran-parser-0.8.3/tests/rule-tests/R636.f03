

! Testing dealloc opt list.
deallocate (this, stat=test)
deallocate (that, stat=test, errmsg=whatever)
end
