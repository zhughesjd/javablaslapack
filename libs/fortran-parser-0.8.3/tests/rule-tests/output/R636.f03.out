1101:begin
629:list-begin
613 this false false
612 1
629:list 1
636:list-begin
613 test false false
612 1
603 false
636:list 1
629:list-begin
613 that false false
612 1
629:list 1
636:list-begin
613 test false false
612 1
603 false
613 whatever false false
612 1
603 false
636:list 2
1103
1101 false true false
